src/main/java/jenkins/branch/buildstrategies/basic/NoTiggerOrganizationFolderMigration.java

```diff
@@ -0,0 +1,49 @@
+package jenkins.branch.buildstrategies.basic;
+
+import hudson.BulkChange;
+import hudson.Extension;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import jenkins.branch.NoTriggerOrganizationFolderProperty;
+import jenkins.branch.OrganizationFolder;
+import jenkins.branch.PropertyMigration;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+@Restricted(NoExternalUse.class)
+@Extension
+public class NoTiggerOrganizationFolderMigration
```

Comment:
> The wonderful thing about tiggers
> Is tiggers are wonderful things!
> Their tops are made out of rubber
> Their bottoms are made out of springs!
> They're bouncy, trouncy, flouncy, pouncy
> Fun, fun, fun, fun, fun!

So you can see why it is essential to exclude them from any involvement in the migration ;-)

(Thankfully it can safely be renamed because the migration implementation  classname is never used anywhere)